### PR TITLE
db : init with v1.0

### PR DESCRIPTION
**Pull request by SupanutT**
This is the original schema from provided database docs. 
I have added some fields which are: job.budget and job.numWorker for ability to set the budget and number of nisit for a particular task.
@mharnsingh is responsible for creating the seeding file in this version 1.0.

### CODE REVIEW

**Review by LittleNa1000 — COMMENTED** [review]
มีแก้นิดหน่อย

File: prisma/seed.ts [hunk 1]
```diff
@@ -0,0 +1,303 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+async function main() {
+
+  await prisma.job.deleteMany({})
+  await prisma.employer.deleteMany({})
+  await prisma.student.deleteMany({})
+  await prisma.user.deleteMany({})
```
Comment (LittleNa1000) [inline prisma/seed.ts:10]: ใช้ Promise.all() ได้นะ

**Review by Fei-7 — APPROVED** [review]
Looks fine to me. Let's go!

File: prisma/seed.ts (hunk 1 above)
Comment (SupanutT) [reply prisma/seed.ts]: done and tested

**Pull request merged**